collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java

```diff
@@ -43,28 +43,13 @@
   @Test
   public void traceSinkGetsCalled() throws InterruptedException
   {
-    this.restTemplate.postForLocation("/api/v1/spans", createTestSpans(1l, 2l, 45l));
-    verify(traceSink, times(3)).receive(any(Trace.class));
-    verifyNoMoreInteractions(traceSink);
-  }
+    //LOTS_OF_SPANS is too much! I need at most 10!
+    final Span[] spans = Arrays.copyOfRange(LOTS_OF_SPANS, 0, min(10, LOTS_OF_SPANS.length));
+    final byte[] spanBytes = Codec.JSON.writeSpans(asList(spans));
```

Comment:
TestObjects.TRACE has three spans that belong to the same trace, TraceTranslator#translateSpans squashes them into one trace and invokes traceSink#receive just once. I would rather keep 10 separate disconnected spans, at least it will make verify block more or less meaningful.